stonesoup/models/transition/tests/test_ct.py

```diff
@@ -43,24 +44,24 @@ def base(model, state_vec, noise_diff_coeffs, turn_rate):
              [0, sp.sin(turn_ratedt),
               0, sp.cos(turn_ratedt)]])
 
-    qx = noise_diff_coeffs[0]
-    qy = noise_diff_coeffs[1]
-    Q = sp.array([[sp.power(qx, 2) * sp.power(timediff, 3) / 3,
```

Comment:
I assume `sp.array` will not merge well here, as `master` is now using `np.array`

This applies throughout

Reply:
I can take care of this when merging.